src/components/MultiselectTags/MultiselectTags.vue

```diff
@@ -0,0 +1,160 @@
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+## Single tag selector
+
+```vue
+<template>
+	<div class="wrapper">
+		<MultiselectTags v-model="value" :multiple="false" />
+		{{ value }}
+	</div>
+</template>
+
+<script>
+export default {
+	data() {
+		return {
+			value: 1
+		}
+	}
+}
+</script>
+```
+
+## Multiple tag selector
+```vue
+<template>
+	<div class="wrapper">
+		<MultiselectTags v-model="value" :multiple="true" />
+		{{ value }}
+	</div>
+</template>
+
+<script>
+export default {
+	data() {
+		return {
+			value: [1, 2]
+		}
+	}
+}
+</script>
+```
+</docs>
+
+<template>
+	<Multiselect :value="inputValue"
+		:options="tags"
+		:options-limit="5"
+		:placeholder="label"
+		track-by="id"
+		:custom-label="tagLabel"
+		:multiple="multiple"
+		:close-on-select="multiple"
+		:tag-width="60"
+		:disabled="disabled"
+		@input="update">
+		<span slot="noResult">{{ t('core', 'No results') }}</span>
+		<template #option="scope">
+			{{ tagLabel(scope.option) }}
+		</template>
+	</multiselect>
+</template>
+
+<script>
+import { Multiselect } from 'Components/Multiselect'
+import { searchTags } from './api'
+
+export default {
+	name: 'MultiselectTags',
+	components: {
+		Multiselect
+	},
+	props: {
+		label: {
+			type: String,
+			default: t('systemtags', 'Select a tag')
+		},
+		// eslint-disable-next-line
+		value: {},
+		disabled: {
+			type: Boolean,
+			default: false
+		},
+		multiple: {
+			type: Boolean,
+			default: true
+		}
+	},
+	data() {
+		return {
+			tags: []
+		}
+	},
+	computed: {
+		inputValue() {
+			return this.getValueObject()
+		}
+	},
+	beforeCreate: function() {
+		searchTags().then((result) => {
+			this.tags = result
+		}).catch(console.error.bind(this))
+	},
+	methods: {
+		getValueObject() {
+			if (this.tags.length === 0) {
+				return []
+			}
+			if (this.multiple) {
+				return this.value.filter((tag) => tag !== '').map(
+					(id) => this.tags.find((tag2) => tag2.id === '' + id)
+				)
+			} else {
+				return this.tags.find((tag) => tag.id === this.value)
+			}
+		},
+		update(value) {
+			if (this.multiple) {
+				this.$emit('input', value.map((element) => element.id))
+			} else {
+				if (value === null) {
+					this.$emit('input', '')
+				} else {
+					this.$emit('input', value.id)
+				}
+			}
+		},
+		tagLabel({ displayName, userVisible, userAssignable }) {
+			if (userVisible === false) {
+				return t('systemtags', '%s (invisible)').replace('%s', displayName)
```

Comment:
Why not use the direct t(app,string,data) function?
```suggestion
				return t('systemtags', '{displayName} (invisible)', { displayName })
```

Reply:
Because the translation is like that in the systemtags app :see_no_evil: